.gitpod.yml

```diff
@@ -1,3 +1,10 @@
 tasks:
-  - init: npm install && npm run build
+  - command: gp await-port 3000 && sleep 3 && gp preview $(gp url 3000)
+  - init: npm install
     command: npm run dev
+ports:
+  - port: 3000
+    onOpen: ignore
+vscode:
+  extensions:
+    - sysoev.language-stylus@1.11.0:xX39oruAJ5UQzTNVRdbBaQ==
```

Comment:
do we need to add the `.vsix` file for this in gitpod ?

Reply:
Thanks! No this would do the job and it'll install the extension under the hood:

```
vscode:
  extensions:
    - sysoev.language-stylus@1.11.0:xX39oruAJ5UQzTNVRdbBaQ==
```

Reply:
cool